packages/create-app/templates/default-app/packages/backend/package.json.hbs

```diff
@@ -33,6 +33,7 @@
     "@backstage/plugin-search-backend-module-pg": "^{{version '@backstage/plugin-search-backend-module-pg'}}",
     "@backstage/plugin-search-backend-node": "^{{version '@backstage/plugin-search-backend-node'}}",
     "@backstage/plugin-techdocs-backend": "^{{version '@backstage/plugin-techdocs-backend'}}",
+    "better-sqlite3": "^7.5.0",
```

Comment:
This should hopefully clean up some of that issue @awanlin too. 👍

Reply:
Actually perhaps I should extract this into a separate change so that we can merge it ahead of the release?

Reply:
#13079